integration_tests/run_pyspark_from_build.sh

```diff
@@ -221,8 +221,8 @@ else
     fi
 
     # Set the Delta log cache size to prevent the driver from caching every Delta log indefinitely
-    export PYSP_TEST_spark_driver_extraJavaOptions="-ea -Duser.timezone=UTC -Ddelta.log.cacheSize=10 $COVERAGE_SUBMIT_FLAGS"
-    export PYSP_TEST_spark_executor_extraJavaOptions='-ea -Duser.timezone=UTC'
+    export PYSP_TEST_spark_driver_extraJavaOptions="-ea -Duser.timezone=UTC -da:ai.rapids.cudf.AssertEmptyNulls -Ddelta.log.cacheSize=10 $COVERAGE_SUBMIT_FLAGS"
+    export PYSP_TEST_spark_executor_extraJavaOptions='-ea -da:ai.rapids.cudf.AssertEmptyNulls -Duser.timezone=UTC'
```

Comment:
Do we have a list of tests that are failing? If so please file issues to fix all of them. If not, then we need a follow on issue to get that list. Turning off the asserts is only a temporary fix.

Reply:
I created an umbrella issue #8183 rather than creating an issue for each test. I haven't run the entire suite of integration tests to know each test but that will be my next task. The only unit test that I have seen fail is md5 as well. If you create an issue please assign it to me

Reply:
I ran a lot of the integration tests. I will file issues for what I find there.